autobisect-js/autoBisect.py

```diff
@@ -687,8 +687,8 @@ def getHgwebMozillaOrg(branchName):
 
 
 def getIdFromTboxUrl(url):
-    """Return numeric ID from treeherder at https://archive.mozilla.org/pub/mozilla.org/firefox/treeherder-builds/ ."""
-    return filter(None, url.split('/'))[-1]
+    """Return numeric ID from treeherder at https://archive.mozilla.org/pub/firefox/tinderbox-builds/ ."""
+    return [i for i in url.split("/") if i is not None][-2]  # There should be a trailing "/"
```

Comment:
Got it. Re. from `[-1]` to `[-2]`, yes it is intentional. (I mildly tested before and after)